### PR TITLE
Add a new task runit

### PR DESCRIPTION
**Pull request by cifren**
Task "runit" could replace the bootRun task until the bug is found and resolve

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -33,3 +33,16 @@ task generateKeys {
...
+task runit {    
+    doLast{
+        javaexec { 
+            main="-jar";
+            args = [
+                "cas/build/libs/cas.war",
+                "-Xmx256M"
+            ]
+        }
+        println '---- Runs and flys (Maybe crashes...) ! ----'
```
Comment (mmoayyed) [inline build.gradle]: What's this part?

File: README.md [hunk 2]
```diff
@@ -104,7 +104,10 @@ java -jar cas/build/libs/cas.war
 Run the CAS web application as an executable WAR via Spring Boot. This is most useful during development and testing.
 
 ```bash
-./gradlew[.bat] bootRun
```
Comment (mmoayyed) [inline README.md:107]: Do not modify this please. Add the alternative to the `Executable WAR` section instead.

File: build.gradle (hunk 1 above)
Comment (cifren) [reply build.gradle]: The println was a bit much maybe...

Comment (cifren) [reply build.gradle]: And Xmx256 limit the memory usage, but it should be removed too.

File: README.md (hunk 2 above)
Comment (cifren) [reply README.md:107]: "./gradlew[.bat] bootRun" still appears, see the added part, I don't know why git specify it as deleted.

**Comment by mmoayyed** [comment]
Thank you!

File: build.gradle (hunk 1 above)
Comment (mmoayyed) [reply build.gradle]: Yup. I was guessing that folks would be less inclined to use/run something if it has the possibility of crashing :)

**Pull request merged**